teku/src/main/java/tech/pegasys/teku/cli/options/P2POptions.java

```diff
@@ -218,6 +223,14 @@ The network interface(s) on which the node listens for P2P communication.
       arity = "0..*")
   private List<String> p2pStaticPeers = new ArrayList<>();
 
+  @Option(
+      names = {"--p2p-static-peers-file"},
+      paramLabel = "<FILENAME>",
+      description =
+          "Specifies a file containing a list of 'static' peers (one per line) with which to establish and maintain connections",
+      arity = "1")
+  private String p2pStaticPeersFile;
```

Comment:
I do like this idea, but potentially we tweak it a little...

We should probably take a `--p2p-static-peers-url` here, and accept a url or file much like we do with initial-state.

Theres helpers in ResourceLoader for loading `urlOrFile` that should help a lot. It's worth getting this right for the feature because changing a documented cli argument is complicated...